Doppler.PushContact.Test/Controllers/PushContactControllerTest.cs

```diff
@@ -483,6 +484,231 @@ public async Task UpdateEmail_should_return_internal_server_error_when_service_t
+
+            var client = _factory.CreateClient(new WebApplicationFactoryClientOptions());
+
+            var request = new HttpRequestMessage(HttpMethod.Put, $"push-contacts/{deviceToken}/visitor-guid")
+            {
+                Headers = { { "Authorization", $"Bearer {token}" } },
+                Content = JsonContent.Create(visitorGuid)
+            };
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData(TestApiUsersData.TOKEN_SUPERUSER_EXPIRE_20010908)]
+        public async Task UpdateOldPushContacts_should_return_unauthorized_when_token_is_a_expired_superuser_token(string token)
+        {
+            // Arrange
+            var fixture = new Fixture();
+
+            var deviceToken = fixture.Create<string>();
+            var visitorGuid = fixture.Create<string>();
+
+            var client = _factory.CreateClient(new WebApplicationFactoryClientOptions());
+
+            var request = new HttpRequestMessage(HttpMethod.Put, $"push-contacts/{deviceToken}/visitor-guid")
+            {
+                Headers = { { "Authorization", $"Bearer {token}" } },
+                Content = JsonContent.Create(visitorGuid)
+            };
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData(TestApiUsersData.TOKEN_EXPIRE_20330518)]
+        [InlineData(TestApiUsersData.TOKEN_SUPERUSER_FALSE_EXPIRE_20330518)]
+        [InlineData(TestApiUsersData.TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518)]
+        public async Task UpdateOldPushContacts_should_require_a_valid_token_with_isSU_flag(string token)
+        {
+            // Arrange
+            var fixture = new Fixture();
+
+            var deviceToken = fixture.Create<string>();
+            var visitorGuid = fixture.Create<string>();
+
+            var client = _factory.CreateClient(new WebApplicationFactoryClientOptions());
+
+            var request = new HttpRequestMessage(HttpMethod.Put, $"push-contacts/{deviceToken}/visitor-guid")
+            {
+                Headers = { { "Authorization", $"Bearer {token}" } },
+                Content = JsonContent.Create(visitorGuid)
+            };
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task UpdateOldPushContacts_should_return_unauthorized_when_authorization_header_is_empty()
```

Comment:
Reemplazaria el nombre de los tests de manera similar a esto: 
```suggestion
        public async Task UpdatePushContactVisitorGuid_should_return_unauthorized_when_authorization_header_is_empty()
```